src/main/java/org/fogbowcloud/saps/engine/core/archiver/storage/nfs/NfsPermanentStorage.java

```diff
@@ -20,31 +26,41 @@
 public class NfsPermanentStorage implements PermanentStorage {
 
     public static final Logger LOGGER = Logger.getLogger(NfsPermanentStorage.class);
-    static final String NFS_STORAGE_TASK_DIR_PATTERN = "%s" + File.separator + "%s" + File.separator + "%s";
+    public static final String NFS_STORAGE_TASK_DIR_PATTERN = "%s" + File.separator + "%s" + File.separator + "%s";
+    public static final String NFS_STORAGE_TASK_DATA_URL_PATTERN = "%s" + File.separator + "%s";
 
     private final String nfsTempStoragePath;
     private final String nfsPermanentStoragePath;
+    private final String nfsPermanentStorageTasksDir;
+    private final String nfsPermanentStorageDebugTasksDir;
```

Comment:
I believe the names of these variables are verbose. I suggest removing the nfs-prefix.

Reply:
Done